std_msgs/Header msg
int32 ID
---
std_msgs/Header msg2
my_package/RVL robotMessage
